src/Service/Startup.cs

```diff
@@ -352,10 +320,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, RuntimeC
                 app.UseDeveloperExceptionPage();
             }
 
-            if (!RuntimeConfigProvider.IsHttpsRedirectionDisabled)
-            {
-                app.UseHttpsRedirection();
-            }
+            app.UseHttpsRedirection();
```

Comment:
`app.UseHttpsRedirection();` should still be gated by a config flag, no?

Reply:
Ah yes, there's a regression I need to fix.